Internship.API/Models/Intern.cs

```diff
@@ -0,0 +1,29 @@
+﻿using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using System;
+
+namespace Intern.Models
+{
+    public class Intern
+    {
+        [BsonId]
+        [BsonRepresentation(BsonType.ObjectId)]
+        [Required]
+        public int UserId { get; set; }
+        [Required]
+        public int MentorId { get; set; }
+        public string TechnologiesSummary { get; set; }
```

Comment:
This property is not string type, it should be "TechnologiesSummary" instead of string